thunder/core/update_aliases.py

```diff
@@ -42,11 +51,15 @@ def _involves_viewed_args(bsym, viewed):
     return any(isinstance(p, TensorProxy) and variableify(p) in viewed for p in bsym.flat_proxy_args)
 
 
-def insert_alias_updates(computation_trace: Trace) -> Trace:
+def insert_alias_updates(computation_trace: Trace, alias_tensor_indices: list[list[int]]) -> Trace:
```

Comment:
Could you give a docstring to this function and then add this function to docs by adding `update_aliases.rst` to https://github.com/Lightning-AI/lightning-thunder/tree/4acf8198cfed1761528f9d10d2bc1a5356cf9ce9/docs/source/reference/core, mayb e as a follow-up? 
It'd be really nice if that docstring mentions the difference between this and functionalization.

Reply:
This is a good idea.  One that deserves its own PR.